MANIFEST.in

```diff
@@ -1,4 +1,2 @@
 include MANIFEST.in
-include *.txt
-include *.md
-include *.cfg
```

Comment:
The `pypi_deploy` CI errors on this PR are caused by the removal of these three lines. Please add back:

```
include *.txt
include *.md
include *.cfg
```